docs/advanced/advanced.md

```diff
@@ -428,3 +428,15 @@ helm repo update
 helm install --create-namespace -n cattle-logging-system rancher-logging-crd rancher-charts/rancher-logging-crd
 helm install --create-namespace -n cattle-logging-system rancher-logging --set additionalLoggingSources.k3s.enabled=true rancher-charts/rancher-logging
 ```
+
+## Additional Network Policy Logging
+
+Dropped packets by network policies can be logged. The log is printed by iptables NFLOG action and shows the packet details and the network policy that bloked it.
+
+To do so, install ulogd2 and configure `[log1]` to read on `group=100`. Then, restart the ulogd2 service for the new config to be committed. 
```

Comment:
```suggestion
To convert the NFLOG packets to log entries, install ulogd2 and configure `[log1]` to read on `group=100`. Then, restart the ulogd2 service for the new config to be committed. 
```

Reply:
For people who don't want to install and configure ulogd2 while troubleshooting, is there an example tcpdump command that can be used to quickly print the packets?

Reply:
Good idea, just added it